.github/workflows/coverage-documentation.yaml

```diff
@@ -46,7 +49,7 @@ jobs:
       - name: Install Rust nightly
         run: |
           rustup toolchain install nightly --profile minimal --target wasm32-unknown-unknown ${{ matrix.mt.component }}
-          rustup default nightly
+          rustup default ${{ matrix.mt.toolchain }}
```

Comment:
Test coverage only works on nightly right now.
This is a limitation by Rust and `wasm-bindgen`.